### ISSUE TITLE
Identify Features cannot seem to be switched off

### ISSUE THREAD

**Issue by MarByteBeep** [qgis/QGIS#46460]
### What is the bug or the crash?

Whenever I select Identify Features (or press CTRL-SHIFT-i) the option remains switched on and there seems to be no way to switch it off, other than restarting the application

### Steps to reproduce the issue

1. Open new project
2. Click "Identify Features" and observe it being switched on
3. Click "Identify Features" again, and observe it *still* being switched on

### Versions

QGIS version
3.22.1-Białowieża
QGIS code revision
663dcf8fb9
Qt version
5.15.2
Python version
3.9.5
GDAL/OGR version
3.4.0
PROJ version
8.2.0
EPSG Registry database version
v10.038 (2021-10-21)
GEOS version
3.10.0-CAPI-1.16.0
SQLite version
3.35.2
PDAL version
2.3.0
PostgreSQL client version
13.0
SpatiaLite version
5.0.1
QWT version
6.1.3
QScintilla2 version
2.11.5
OS version
Windows 10 Version 2009

Active Python plugins
BGTImport
3.16
inspireNL
2.6
QuickOSM
2.0.0
db_manager
0.1.20
grassprovider
2.12.99
MetaSearch
0.3.5
processing
2.12.99
sagaprovider
2.12.99

### Supported QGIS version

- [ ] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

**Comment by antoniolocandro** [comment]
to 'turn it off' you need to click on the little hand icon to switch tool. Did you try that?

**Comment by MarByteBeep** [comment]
Thanks. That helped. But to be honest, that wasn't very intuitive, for a first time user. I expect that when I select a tool, I can also switch it off again using the same input. And because the hand tool isn't close to this features tool, I didn't notice it switched. If this could be improved upon, please keep this issue open. Otherwise feel free to close it. Thanks a lot for your help!

**Comment by JoJ66** [comment]
Hi,

I'm working with  version 3.28.1 on Windows 10. 
I'm having the same issue (and I'm a "newbee" as well).  Yet... the only " little hand icon" I found was the icon Pan Map and now I was stuck in this mode... until I clicked another icon, where I got stuck again. Now I'm stuck in 'Select Feature by Area' function for my cursor.
Conclusion: this issue counts for several icons. 

Do I miss a button in my toolbar, that resets my cursor to plain cursor? Or that switches tool/function and then can be turned off? I would like my cursor to be a plain cursor between actions.
